topics/admin/tutorials/gxadmin/tutorial.md

```diff
@@ -50,9 +50,18 @@ Since then it became the home for "all of the SQL queries we [galaxy admins] run
 >
 > 2. Install the role with `ansible-galaxy install -p roles -r requirements.yml`
 >
-> 3. Add the role to your playbook, it should run as root
+> 3. Add the following variables to your group variables file:
```

Comment:
Mention that maybe you already have them set if you ran the Grafana tutorial first?